src/inmanta/protocol/common.py

```diff
@@ -980,10 +981,10 @@ def attach_warnings(code: int, value: Optional[JsonType], warnings: Optional[Lis
     return code, value
 
 
-def json_encode(value: ReturnTypes) -> str:
+def json_encode(value: ReturnTypes, tz_aware: bool = False) -> str:
```

Comment:
I think it might be better to make this `True` by default (for all methods). Places where you currently don't explicitly set the option (again, for all methods):
- `inmanta.data` -> doesn't really matter here except that if we change it the attribute hash will change but only once. I think this is acceptable but @wouterdb knows best
- `inmanta.agent.handler` -> doesn't matter too much but I think we can safely change it so I think we should
- `inmanta.data.model` -> doesn't matter at all
- `gzipped_json` method 5 lines down from this comment -> if I understand correctly this is what the client uses to send data to the server. I think we can safely change this one
- `inmanta.export` -> doesn't matter too much but again, no reason not to
- `inmanta.resources` -> doesn't matter at all
- `inmanta.protocol.openapi.converter` -> I think it doesn't matter here

Reply:
- `inmanta.data`: should've been more clear: I'm talking about the `Resource.make_hash` method here.

Reply:
@sanderr I don't see how this will affect the attribute hash?

Reply:
Concluded on Slack: shouldn't affect the attribute hash.